plugins/kubernetes/app/views/kubernetes/roles/_form.html.erb

```diff
@@ -18,7 +18,7 @@
       </div>
     <% end %>
 
-    <%= form.input :autoscaled, label: "Replicas managed externally", as: :check_box, help: "Deploying will not change replicas count. Use this when autoscaling deployments." %>
+    <%= form.input :autoscaled, label: "Replicas managed externally", as: :check_box, help: simple_format("- Deploying will not change replicas count.\n- Replica counts be used as the minimum number of replicas that need to be live\n- Use when autoscaling Deployments") %>
```

Comment:
Can't this be an actual HTML list? 

s/Replica counts be used as/Replica counts will be used as

Reply:
done